pkg/podimpersonation/podimpersonation.go

```diff
@@ -559,9 +568,34 @@ func (s *PodImpersonation) augmentPod(pod *v1.Pod, sa *v1.ServiceAccount, secret
 				continue
 			}
 
+			vmount := v1.VolumeMount{
+				Name:      "user-kubeconfig",
+				MountPath: "/tmp/.kube",
+			}
+			cfgVMount := v1.VolumeMount{
+				Name:      "user-kube-configmap",
+				MountPath: "/home/.kube/config",
+				SubPath:   "config",
+			}
+
+			pod.Spec.InitContainers = []v1.Container{
+				{
+					Name:            "init-kubeconfig-volume",
+					Image:           "registry.suse.com/bci/bci-busybox",
```

Comment:
I'm not a fan of using `:latest` tags for images. Something that used to work might stop working randomly if that `:latest` image breaks.

Reply:
Makes sense. Changed it to use `registry.suse.com/bci/bci-busybox:15.5.16.1` which is the latest one available as of today

Reply:
+1 on not using `latest`. Overall we tend to recommend folks tag to the minor: `registry.suse.com/bci/bci-busybox:15.5`. So you latest patches, without the risk of breaking changes.

Reply:
Michael reminded me about airgap. It seems like hardcoding `registry.suse.com/bci/bci-busybox:whatever` would not work for airgap use cases where the customer does not have access to the SUSE registry.

@diogoasouza one idea here would be to re-use the shell image but change its entrypoint via `command:`. That would avoid making the busybox available for airgap use case (no idea if it's already done btw).

Reply:
@tomleb I checked and the busybox image is mirrored, but only the `15.4` version, and that was more than one year ago. I changed the init container to use the shell image, it shouldn't affect the latency since the image is going to have to be pulled anyway for the `proxy` and `shell` containers.